crates/wasm-shrink/src/main.rs

```diff
@@ -0,0 +1,27 @@
+use anyhow::Result;
+use structopt::StructOpt;
+use wasm_shrink::WasmShrink;
+
+fn main() -> Result<()> {
+    let shrink = WasmShrink::from_args();
+
+    let mut config = simplelog::ConfigBuilder::new();
```

Comment:
Ideally I think we'd keep all the various CLI tools consistent, so could the others be switched to `simplelog` or could this use `env_logger`?

Reply:
The thing is that this is using `log::*` macros for *all* output, because I wanted something that would be compatible with either CLI or library usage where you don't necessarily want to write to stdout/stderr, so if the CLI uses `env_logger`, then you get zero output by default, which is pretty confusing for this kind of long-running reduction process.

Reply:
So, I don't think it makes sense to switch other tools to `simplelog` and I don't think it makes sense to switch this tool to `env_logger`.

Reply:
Actually once I split the I/O out from the library and only in the CLI this was pretty easy to resolve!